ios/Classes/PdftronFlutterPlugin.h

```diff
@@ -498,8 +505,8 @@ typedef enum
     zoomChangedId,
     pageMovedId,
     scrollChangedId,
-
     // Hygen Generated Event Listeners (2)
+    annotationToolbarItemPressedId,
```

Comment:
The order of these enum values (all 15 of them) must __exactly__ match the order of these ones in Dart:
https://github.com/PDFTron/pdftron-flutter/blob/master/lib/src/events.dart#L136-L155

Specifically, the `annotationToolbarItemPressedId` enum member should come before the `scrollChangedId` member.

Reply:
fixed ordering of this enum